modules/services/window-managers/i3-sway/lib/options.nix

```
@@ -831,4 +831,33 @@ in {
     '';
     example = "workspace number 9";
   };
+
+  workspaceOutputAssign = mkOption {
+    type = with types;
+      let
+        workspaceOutputOpts = submodule {
+          options = {
+            workspace = mkOption {
+              type = str;
+              default = "";
+              example = "Web";
+              description = ''
+                Name of the workspace to assign.
+              '';
+            };
+
+            output = mkOption {
+              type = str;
+              default = "";
+              example = "eDP";
+              description = ''
+                Name of the output from `xrandr --current`.
```

Does xrandr work for sway? Either way, the more native way is probably


```suggestion
                Name of the output from <command>swaymsg -t get_outputs</command>.
```

Also note that AFAIK we don't use Markdown in option documentation.

---

Changed with switch for i3-msg and swaymsg like in e.g. defaultWorkspace :)